provisioning/AWS/aws-php-storage-driver-common-resources.yaml

```diff
@@ -51,17 +47,6 @@ Resources:
                   - arn:aws:s3:::keboola-drivers/*
                   - arn:aws:s3:::keboola-drivers
                 Effect: Allow
-        - PolicyName: FileStorage
-          PolicyDocument:
-            Version: "2012-10-17"
-            Statement:
-              - Action:
-                  - s3:ListBucket
-                  - s3:GetObject
-                Resource:
-                  - arn:aws:s3:::ci-storage-driver-common/*
```

Comment:
Pokud ta role mela mit pravo pro pristup k tomu bucketu tak tohle je spravny pristup. Jenom misto hardcoded jmena tu zareferencujes pomoci `Ref` vytvoreny bucket

Reply:
Jenom jak jsme se bavili by pro import/export testy ktere k bucketu pristupuji by bylo dobry vytvorit dalsi roli

Reply:
Jo to som skusall a nejak mi to nefungovalo a neviem preco, vyskusam to este zajtra a ked tak sa ozvem len jedna otazka malo by to pouzivat rovnaky identity provider pre obe role, ze ? (nejde vytvorit dalsi pre token.actions.githubusercontent.com).

Reply:
jo jasne identity provider jeden. Dej vedet a mrkneme na to.

Reply:
Ty jo chova sa to nejak divne, ak by si mal cas niekedy po 9 si nad tim zavolat. 

1. https://github.com/keboola/storage-backend/actions/runs/4279721888 tu to preslo
2. potom som len skusal zmazat arg co su tam na viac a spadlo to 
3. tak som to skusil vratit do bodu 1 a uz to vobec nepreslo.

Reply:
V 9 mame standup tak asi nejak potom, nebo klidne muzu ted.